transcendence-app/src/user/user.controller.ts

```diff
@@ -30,10 +38,15 @@ export class UserController {
     return this.userService.findOneOrCreate(query);
   }
 
+  @Get('me')
+  getCurrentUser(@GetRequest() req: Request) {
+    return req.user;
+  }
+
```

Comment:
Despite this issue was assigned to me , very nice job. 😄

Reply:
Thank you! I've added this endpoint to test if sessions are working correctly.